### DISCUSSION TITLE
Get gamestate right after your snake died

### DISCUSSION THREAD

**Discussion by hkstests** [BattlesnakeOfficial/feedback#183] [Q&A] [answered]
Hi all,

I just noticed that you dont get any /Move requests after your snake dropped out (which kinda makes sense :D ).
So, is there any way to get the entire gamestate right after your snake died?
I am only aware of the content in the /End request which unfortunately only includes the data of your own snake when it died.

**Comment by bvanvugt** [comment, marked as answer]
This is a neat idea, and I totally get the motivation. Not something we're going to change mid-league, but curious to hear feedback and community thoughts.
